src/client/App.js

```diff
@@ -49,6 +51,12 @@ function App() {
         <Route exact path="/contact-us">
           <ContactUsPage />
         </Route>
+         {/* this is set here just for testing,will be moved to authentication route */}  
```

Comment:
If you format like a todo comment like this, it is easy to get a list of todos in Vscode :)

```suggestion
         {/* TODO: this is set here just for testing,will be moved to authentication route */}  
```

Reply:
sure